src/content/docs/design/design-with-code/connect-repo.mdx

```diff
@@ -0,0 +1,22 @@
+---
+page_id: 261fb4ed-83e5-4f92-8f52-8da22749a92e
+title: Connect your repo for custom pages
+sidebar:
+  order: 2
+relatedArticles:
+  - 66f83c18-d261-48b8-a517-648ed1b7b064
+  - b42663d8-5e2a-42cd-859a-f9737a1495df
+  - 743d264d-ef08-4f2e-a42c-2b971152dccf
+---
+
+To enable your design code to override Kinde page design, you need to set up and connect a git repo for the design files. You must also be using a [custom domain](/build/domains/pointing-your-domain/) to run your custom design code with Kinde.
+
+1. Go to **Design > Custom code**.
+2. Select the option to **Connect repo**. This will take you to the **Settings > Git Repo** page where you can manage your repo connection for design and workflows.
+3. If you don’t already have a repo set up for workflows, select **Connect GitHub**. 
+4. Follow the prompts to choose a repo and branch. 
+5. Once connected, choose if you want to **set synced code live immediately**. This is more efficient for dev and testing environments, but you might switch this off in your production environment.
+
+### Set up the GitHub directory structure
+
+Your directory structure should be in the format `pages/(kinde)/(<route>)/page.[ts/tsx]` where `<route>` is used to map your code to a rendered page in Kinde. For example `pages/(kinde)/(register)/page.tsx` will map to the register / sign up page in the authentication flow. For more information, see [Understand page customization](/design/design-with-code/understand-page-design/).
```

Comment:
@clairekinde11 I added a block next to one of @chris-pearce's comments in the doc which gave a directory tree - that would sit quite well in this section